tomcat/src/main/java/org/apache/coyote/http11/HttpRequest.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.coyote.http11;
+
+import java.util.Map;
+
+public class HttpRequest {
+
+    private final Method method;
+    private final String path;
+    private final HttpVersion httpVersion;
+    private final ContentType contentType;
+    private final Map<String, String> queryParameter;
+
+    public HttpRequest(Method method,
+                       String path,
+                       HttpVersion httpVersion,
+                       ContentType contentType,
+                       Map<String, String> queryParameter) {
+        this.method = method;
+        this.path = path;
+        this.httpVersion = httpVersion;
+        this.contentType = contentType;
+        this.queryParameter = queryParameter;
+    }
+
+    public Method getMethod() {
+        return method;
+    }
+
+    public String getPath() {
+        return path;
+    }
+
+    public ContentType getContentType() {
+        return contentType;
+    }
+
+    public String getQueryParameterValue(String key) {
+        String value = queryParameter.get(key);
+        if (value == null || value.isBlank()) {
+            return "";
+        }
+        return value;
```

Comment:
value가 "   " 처럼 공백 문자로 채워진 경우 "" 응답을 기대하지 않았다면
다음 처럼 getOrDefault를 사용해도 좋을 것 같아요.
```suggestion
        return queryParameter.getOrDefault(key, "");
}
```